doc/adv-manual/asset-site-association.rst

```diff
@@ -1,34 +1,31 @@
 How the hazard sites are determined
-=====================================================
+===================================
 
 There are several ways to specify the hazard sites in an engine calculation.
-There are listed here, in order of precedence.
 
-1. The user can specify the sites directly in the job.ini
-   (i.e. `sites = 0 0, 1 2`) This is the most direct way.
-2. The user can specify the sites in a .csv file
+1. The user can specify the sites directly in the job.ini using the `sites`
+   parameter (e.g. `sites = -122.4194 37.7749, -118.2437 34.0522, -117.1611 32.7157`).
+   This method is perhaps most useful when the analysis is limited to a 
+   handful of sites.
+2. Otherwise the user can specify the list of sites in a CSV file
    (i.e. `sites_csv = sites.csv`).
-3. The user can specify a grid via the `region` and
-   `region_grid_spacing` parameters
-4. If the `region` parameter is
-   missing but `region_grid_spacing` is specified and there is an exposure,
-   a grid is implicitly generated from the convex hull of the exposure
-5. Otherwise the sites can be inferred from the site model, if any.
-6. If the user is running a `classical_risk` or
-   `classical_damage` calculation starting from a set of hazard curves
-   in CSV format, then the sites are read from such file.
-7. Finally, the sites can be extracted from the exposure.
-
-It must be noticed that the engine rounds longitudes and latitudes
-to 5 digits (1 meter resolution) so sites that differs at the 6 digit or so
-end up as being duplicated for the engine, and they may give issues.
+3. Otherwise the user can specify a grid via the `region` and
+   `region_grid_spacing` parameters.
+4. Otherwise the sites can be inferred from the exposure, if any,
+   in two different ways:
    
+   4a. if `region_grid_spacing` is specified, a grid is implicitely
```

Comment:
You removed the part about longitude and latitude rounding, which is quite important.

Reply:
Inadvertently, adding it back.